tests/error-display/error-display.spec.tsx

```diff
@@ -62,173 +65,67 @@ describe("ErrorDisplay", () => {
-            ).toBe(
-                transformJSXElementToString(error.description as JSX.Element)
-            );
-        });
-    });
-
-    describe("408 error", () => {
-        it("should be able to render", () => {
-            const type = "408";
-            render(<ErrorDisplay type={type} />);
-
-            const error = ERROR_DISPLAY_DATA.get(type);
-            expect(
-                screen.getByRole("heading", { level: 1, name: error.title })
-            ).toBeInTheDocument();
-
-            expect(screen.getByRole("img")).toHaveAttribute(
-                "src",
-                error.img.src
-            );
-            expect(
-                screen.getByTestId(ERROR_DESCRIPTION_TEST_ID).textContent
-            ).toBe(
-                transformJSXElementToString(error.description as JSX.Element)
-            );
-        });
-    });
-
-    describe("500 error", () => {
-        it("should be able to render", () => {
-            const type = "500";
-            render(<ErrorDisplay type={type} />);
-
-            const error = ERROR_DISPLAY_DATA.get(type);
-            expect(
-                screen.getByRole("heading", { level: 1, name: error.title })
-            ).toBeInTheDocument();
-
-            expect(screen.getByRole("img")).toHaveAttribute(
-                "src",
-                error.img.src
-            );
-            expect(
-                screen.getByTestId(ERROR_DESCRIPTION_TEST_ID).textContent
-            ).toBe(
-                transformJSXElementToString(error.description as JSX.Element)
-            );
-        });
-
-        it("should be able to render reference ID", () => {
-            const type = "500";
-            const additionalProps = { referenceId: "123" };
-
-            render(
-                <ErrorDisplay type={type} additionalProps={additionalProps} />
-            );
-
-            const error = ERROR_DISPLAY_DATA.get(type);
-            const errorDescription = transformJSXElementToString(
-                error.renderDescription(additionalProps) as JSX.Element
-            );
-
-            expect(
-                screen.getByTestId(ERROR_DESCRIPTION_TEST_ID).textContent
-            ).toBe(errorDescription);
-        });
-    });
-
-    describe("503 error", () => {
-        it("should be able to render", () => {
-            const type = "503";
-            render(<ErrorDisplay type={type} />);
-
-            const error = ERROR_DISPLAY_DATA.get(type);
-            expect(
-                screen.getByRole("heading", { level: 1, name: error.title })
-            ).toBeInTheDocument();
-
-            expect(screen.getByRole("img")).toHaveAttribute(
-                "src",
-                error.img.src
-            );
-            expect(
-                screen.getByTestId(ERROR_DESCRIPTION_TEST_ID).textContent
-            ).toBe(error.description);
-        });
+    describe("ErrorDisplay components", () => {
+        const testData = [
```

Comment:
should be able to get error from `ERROR_DISPLAY_DATA.get(type)` so there's no need to specify it in `testData`

then `testData` can be declared as `ErrorDisplayType[]`

Reply:
arh.. my bad. ok!